src/software/jetson_nano/ansible/playbooks/deploy_nano.yml

```diff
@@ -71,6 +71,7 @@
           register: result
           args:
             chdir: ~/thunderbots_binaries
+          ignore_errors: yes
```

Comment:
Do we not want to add `when: "'stop' in actions"`?

Reply:
added